IoTEdgeModbus/Modules/Azure.IoT.Edge.Modbus.Tests/Data/Int16DecoderTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Azure.IoT.Edge.Modbus.Tests.Data
+{
+    using System;
+    using System.Linq;
+    using AzureIoTEdgeModbus.Slave;
+    using AzureIoTEdgeModbus.Slave.Data;
+    using AzureIoTEdgeModbus.Slave.Decoding;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+    [TestClass]
+    public class Int16DecoderTests
+    {
+        [TestMethod]
+        public void CanDecodeValue()
```

Comment:
Should this be using DataTestMethod as well like other tests?

Reply:
Remove unused namespace declarations please.

Reply:
Changed to [DataTestMethod].
Removed unused using statements.